.env

```diff
@@ -1,11 +1,16 @@
 # Proxy urls
 VITE_USE_PROXYURL=true
-VITE_PROXYURL_WMS="https://map.geoportail.lu/ogcproxywms"
-VITE_PROXYURL_REMOTE="https://map.geoportail.lu/httpsproxy"
+VITE_PROXYURL_WMS="/ogcproxywms"
+VITE_PROXYURL_REMOTE="/httpsproxy"
+
+# Urls for metadata, metadata links and legends
+VITE_GEONETWORK_URL="https://geocatalogue.geoportail.lu/geonetwork/srv"
+VITE_GET_LEGENDS_URL="/legends/get_html"
+VITE_GET_METADATA_URL="/getMetadata"
```

Comment:
I think you forgot to (commit to) use these URLs in the [`layer-metadata.service.ts`](https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/blob/main/src/services/layer-metadata/layer-metadata.service.ts#L15-L20).

Reply:
oops thx for noticing this, I have mixed two PR at some point https://github.com/Geoportail-Luxembourg/luxembourg-geoportail/compare/0e4574ba1485fea33d8b84987db57da07d58a652..512c9e9ea17e54d465499be8f7da45c4d9622355 and forgot to commit the right files.